src/foam/u2/view/ReciprocalSearch.js

```diff
@@ -150,6 +150,10 @@ foam.CLASS({
       var self = this;
 
       this.dao.on.sub(this.updateTotalCount);
+      this.dao.on.sub(function() {
```

Comment:
selectedCount wasn't updated on clearAll so I thought maybe this change will make some sense